numpy/lib/mixins.py

```diff
@@ -152,9 +152,6 @@ def __repr__(self):
     __mul__, __rmul__, __imul__ = _numeric_methods(um.multiply, 'mul')
     __matmul__, __rmatmul__, __imatmul__ = _numeric_methods(
         um.matmul, 'matmul')
-    if sys.version_info.major < 3:
-        # Python 3 uses only __truediv__ and __floordiv__
-        __div__, __rdiv__, __idiv__ = _numeric_methods(um.divide, 'div')
```

Comment:
if could be useful to add a comment here:
```# Python 3 does not use __div__, __rdiv__, or __idiv__```

Reply:
Done